internal/backends/winit/renderer/femtovg.rs

```diff
@@ -424,17 +423,17 @@ impl FemtoVGCanvas {
         };
         callback(api)
     }
-
-    fn release_graphics_resources(&self) {
-        self.graphics_cache.clear_all();
-        self.texture_cache.borrow_mut().clear();
-    }
 }
 
 impl Drop for FemtoVGCanvas {
     fn drop(&mut self) {
         // The GL renderer must be destructed with a GL context current, in order to clean up correctly.
         self.opengl_context.make_current();
+
+        // Clear these manually to drop any Rc<Canvas>.
+        self.graphics_cache.clear_all();
+        self.texture_cache.borrow_mut().clear();
+
         if Rc::strong_count(&self.canvas) != 1 {
```

Comment:
Is the canvas still functional after the caches are dropped? If not, then this could be an assert_eq! instead: It will crash anyway if somebody tries to use this after we are done...